config/jobs/kubernetes/sig-azure/sig-azure-config.yaml

```diff
@@ -0,0 +1,81 @@
+presets:
+- labels:
+    preset-azure-cred: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: REGISTRY
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_USERNAME
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_PASSWORD_FILE
+    value: /etc/docker-cred/dockercred
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  - name: docker-cred
+    secret:
+      secretName: docker-cred
+  volumeMounts:
+  - name: azure-cred
+    mountPath: /etc/azure-cred
+    readOnly: true
+  - name: azure-ssh
+    mountPath: /etc/azure-ssh
+    readOnly: true
+  - name: docker-cred
+    mountPath: /etc/docker-cred
+    readOnly: true
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure
+    always_run: false
+    branches:
+    - master
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: ritazh/kubekins-e2e:v20190126-731395d-dirty-master # update this after test-infra PRs are merged 
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--root=/go/src"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--repo=k8s.io/kubernetes=$(PULL_REFS)"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=460"
+        - "--scenario=kubernetes_e2e"
+        - --
+        - "--gce-ssh=" # https://github.com/kubernetes/test-infra/pull/10054
+        - "--test=true"
+        - "--up=true"
+        - "--down=true"
+        - "--deployment=acsengine"
+        - "--build=bazel" # build kubectl binary
+        - "--provider=skeleton" # noop
+        - "--ginkgo-parallel=30"
+        - "--acsengine-agentpoolcount=3"
+        - "--acsengine-admin-username=azureuser"
+        - "--acsengine-admin-password=AdminPassw0rd"
+        - "--acsengine-creds=$AZURE_CREDENTIALS" 
+        - "--acsengine-orchestratorRelease=1.12"
```

Comment:
since this is for pull requests, I think we should always use latest orchestrator release here?

Reply:
Unfortunately aks-engine doesn't allow you to specify an orchestrator release of 1.14, and doesn't have a version defined for dev work ( AFAIK ). The option here is to just say orchestrator 1.14 and replace the hyperkube image trough the template with the one built by the job.